src/_backend_agg.h

```diff
@@ -406,6 +406,11 @@ RendererAgg::_draw_path(path_t &path, bool has_clippath, const facepair_t &face,
         double linewidth = points_to_pixels(gc.linewidth);
         if (!gc.isaa) {
             linewidth = (linewidth < 0.5) ? 0.5 : mpl_round(linewidth);
+        } else {
+            // Too small linewidths are clipped because they require a lot of
+            // computation and do not make sense even with antialiasing.
+            // Compare github #14498.
+            linewidth = (linewidth < 0.01) ? 0.01 : mpl_round(linewidth);
```

Comment:
I guess clipping at 1/256 would make sense because the quantization of the channels is 8bit so linewidths<1/256 (or half that? perhaps...) would be quantized to zero.

Reply:
What are channels in this context? I only know color channels but don't see how that could be related to linewidth.

Anyways, 0.01 causes some image comparison failures, trying 0.001 to see if that fixes it.

Reply:
I meant color channels.
With antialiasing, linewidths<1 are basically represented by setting the color channels to (linewidth*real_color) (blending with the background), but that's then quantized to 8 bits so linewidths<1/(2*256) should result in nothing being drawn.